.github/workflows/performance-benchmarks-naslib.yml

```diff
@@ -0,0 +1,197 @@
+name: Performance Benchmarks with naslib
+
+on:
+  workflow_dispatch:
+    inputs:
+      sampler-list:
+        description: 'Sampler List: A list of samplers to check the performance. Should be a whitespace-separated list of Optuna samplers. Each sampler must exist under `optuna.samplers` or `optuna.integration`.'
+        required: false
+        default: 'RandomSampler TPESampler'
+      sampler-kwargs-list:
+        description: 'Sampler Arguments List: A list of sampler keyword arguments. Should be a whitespace-separated list of json format dictionaries.'
+        required: false
+        default: '{} {\"multivariate\":true,\"constant_liar\":true}'
+      pruner-list:
+        description: 'Pruner List: A list of pruners to check the performance. Should be a whitespace-separated list of Optuna pruners. Each pruner must exist under `optuna.pruners`.'
+        required: false
+        default: 'NopPruner'
+      pruner-kwargs-list:
+        description: 'Pruner Arguments List: A list of pruner keyword arguments. Should be a whitespace-separated list of json format dictionaries.'
+        required: false
+        default: '{}'
+      n-runs:
+        description: 'Number of Studies'
+        required: false
+        default: '10'
+
+
+jobs:
+  performance-benchmarks-naslib:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Setup Python3.9
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.9
```

Comment:
NASLib does not support 3.9 officially. 
![image](https://user-images.githubusercontent.com/3255979/164607159-0721aa0b-3eda-438b-aa9e-838ab8653b15.png)

```suggestion
    - name: Setup Python3.8
      uses: actions/setup-python@v2
      with:
        python-version: 3.8
```